### DISCUSSION TITLE
Creating my own theme

### DISCUSSION THREAD

**Discussion by tomron** [Data-Centric-AI-Community/fg-data-profiling#958] [Ideas]
**Missing functionality**

I would like to create my own theme (basically control the buttons and navbar color). How is it possible?

One possible way I can think of to implement it is to pass this as configuration parameters and then generate a CSS file as part of the report generating process.
Another way is to have a command line to create this CSS configuration and pass this file as part of the configuration.

**Comment by sbrugman** [comment]
Several paramaters allow for control of the theme (primary color, logo): https://pandas-profiling.ydata.ai/docs/master/rtd/pages/advanced_usage.html#the-html-report.

The theme itself for the HTML reports is set here: 
https://github.com/ydataai/pandas-profiling/blob/develop/src/pandas_profiling/report/presentation/flavours/html/templates/wrapper/style.html

There are multiple paths to achieve the desired effect. You could modify the report after its generation, if your need is specific to one style/theme. Alternatively, you could modify the source code to output your theme directly. Finally, you could consider contributing a more general feature that allows for custom CSS.
